test/src/test/kotlin/graphql/nadel/tests/TestFixtureUpdater.kt

```diff
@@ -0,0 +1,70 @@
+package graphql.nadel.tests
+
+import com.fasterxml.jackson.module.kotlin.readValue
+import graphql.nadel.tests.yaml.YamlComment.Position.Block
+import graphql.nadel.tests.yaml.YamlComment.Position.End
+import graphql.nadel.tests.yaml.YamlComment.Position.Inline
+import graphql.nadel.tests.yaml.collectComments
+import graphql.nadel.tests.yaml.traverseYaml
+import graphql.nadel.tests.yaml.writeYamlTo
+import java.io.File
+
+// For autocomplete navigation
+private typealias TestFixtureUpdater = Unit
+
+fun main() {
+    val path = ClassLoader.getSystemClassLoader().getResource("")!!.path
+    val projectDir = path.substringBeforeLast("test/build/")
+
+    val fixturesDir = File(projectDir, "test/src/test/resources/fixtures")
+
+    update(fixturesDir)
+}
+
+private fun update(fixturesDir: File) {
+    val yamlExtensions = setOf("yaml", "yml")
+
+    fixturesDir.walkTopDown()
+        .filter {
+            it.extension in yamlExtensions
+        }
+        .forEach { file ->
+            println(file.name)
+
+            // If you don't have trimmed line ends then the writer seems to not want to spit out multi line strings
+            val cleanFileContents = file
+                .readLines()
+                .joinToString(separator = "\n", transform = String::trimEnd)
+
+            val testFixture = yamlObjectMapper.readValue<TestFixture>(cleanFileContents)
+
+            val newFixture = update(testFixture)
+            writeOut(file, newFixture)
+        }
+}
+
+fun update(testFixture: TestFixture): TestFixture {
```

Comment:
maybe move this up to the top and then put lots of comments on it that this is place to add the transformation code.

I guess its always a bit temporary - eg we write it, dump them, then kill the code later.

Maybe have a single class thats generally empty and there to be the transform step only.  Cleaner place t change - meanwhile all the YAML accounting is done in this class

Reply:
Yeah I agree with moving it to the top and adding comments on the process.

I didn't want to add a class etc because I feel like that's just overengineering a script. I think we don't need to do any wacky shit or keep lots of complex code for long, so it's fine (for now).

Yea like you said, we'll write code, then probably kill it off later.